canal/rows.go

```diff
@@ -53,6 +53,9 @@ func (r *RowsEvent) handleUnsigned() {
 
 	for i := 0; i < len(r.Rows); i++ {
 		for _, columnIdx := range r.Table.UnsignedColumns {
+			if columnIdx >= len(r.Rows[i]) {
```

Comment:
thanks @chrisxrepo 

can you show the example how can we meet this?

Reply:
1. start sync binlog
2. stop sync binlog， save binlog position
3.change table schema， example add column
4.start sync binlog from last postion
then panic...

Reply:
I guess the problem is that Canel has not supported DDL change, I think we need to update `r.Table.UnsignedColumns` in handing DDL event.

Reply:
use old binlog data, and use newest table schema, it will case panic

Reply:
Now I have the same problem